package.json

```diff
@@ -187,7 +187,7 @@
     "nyc": "nyc ava \"dist/tests/lib/rules/cloudinary/**/*.js\" --concurrency=2 --timeout=2m --debug",
     "preparecommitmsg": "node scripts/prepare-commit-message.js",
     "release": "npm i && npm run test -- --concurrency 2 --fail-fast && node dist/scripts/release.js",
-    "site": "node dist/src/bin/sonarwhal",
+    "sonarwhal": "node dist/src/bin/sonarwhal",
```

Comment:
:+1:

Did you updated this everywhere in to docs?

Reply:
Now yes